fix: outputEncoding is deprecated in three >= 0.152

squishyu opened the pull request:
Simple import/name change

Fixes #742

usefulthink commented:
Since the default was changed to sRGB (https://github.com/mrdoob/three.js/pull/25783) with the same release, the encoding can be dropped completely for three.js versions 152 and above.

I think this library should still keep support for older three.js versions for some time, so a check with `THREE.REVISION` would be best (so something like `if (REVISION < 152) { renderer.outputEncoding = sRGBEncoding; }`).

squishyu commented:
Added revision check for backwards compatibility and removed outputColorSpace.

```ts
// Since r152, default outputColorSpace is SRGB
// Deprecated outputEncoding kept for backwards compatibility
if (Number(REVISION) < 152) this.renderer.outputEncoding = sRGBEncoding;
```

usefulthink reviewed (approved):
Thanks a lot! Looks good to me.
